src/command.ts

```diff
@@ -141,8 +151,74 @@ const createHandoverCommand = (
       }
     })
 
+  const historyCmd = new CliCommand('history')
+    .withDescription('Fetch a list of previous handover posts')
+    .withOptions({
+      name: ['-d', '--daysBefore'],
+      description: 'Number of days before to fetch handover posts',
+      defaultValue: 7,
+      args: [
+        {
+          name: 'daysBefore',
+          required: false,
+        },
+      ],
+    })
+    .withHelpHandler(showHelp)
+    .withHandler(async (_args, options) => {
+      const { daysBefore } = options
+      const daysBeforeInNumber = parseInt(daysBefore, 10)
+
+      if (daysBeforeInNumber > 30) {
+        await publishPrivateContentToSlack({
+          web,
+          userId,
+          text: "You're pushing it, human. We can't fetch more than 30 days of history!",
+        })
```

Comment:
It's just an arbitrary number. I just thought it would be safer to set a limit for now.

Reply:
Good thinking! A month seems like a reasonable limit